README.md

```diff
@@ -0,0 +1,82 @@
+# Configurable LTI Consumer XBlock
+
+This XBlock is used to replace edX's lti_consumer xblock. It makes available to
```

Comment:
maybe add a link to the original xblock's github repository?
https://github.com/edx/xblock-lti-consumer